content/javascript/concepts/dates/terms/getTime/getTime.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '.getTime()' 
+Description: 'Returns the number of milliseconds since 1st January 1970.' 
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Methods'
+CatalogContent: 
+  - 'path/web-development'
+  - 'introduction-to-javascript'
+---
+
+The **`getTime()`** method returns the number of milliseconds since the 1st of January 1970.
+
+## Syntax
+
+```js
+myDate.getTime()
+```
+
+The returned value from `getTime()` is an number value showing the number of milliseconds since the 1st January 1970.
+
+## Example
+
+In the example the variable `today` stores the number of milliseconds from 1/1/1970 to the current time created using by the `getTime()` method on a new `Date` and logs the result to the console.
+
+```js
```

Comment:
It may be beneficial to adjust the example to use a specific date, so that there is a fixed result, and include a shell block with the output.

Reply:
I've updated the example as suggested but I don't know what shell block means? Is that a codebyte option? @caupolicandiaz

Reply:
@cguttweb It's just a code block like this:
````
```shell
output
```
````